Testreglar/2.2.2/Nett/nett-2.2.2a.json

```diff
@@ -0,0 +1,328 @@
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.6"
+				},
+				"nei": {
+					"type": "gaaTil",
+					"steg": "3.10"
+				}
+			}
+		},
+		{
+			"stegnr": "3.6",
+			"spm": "Ligg mekanismen i direkte tilknytning til innhaldet som beveger seg, blinkar, rullar?",
+			"ht": "<p>Plasseringa kan for eksempel vere rett over eller rett ved sida av innhaldet.</p>",
+			"type": "jaNei",
+			"kilde": [
+				"G186"
+			],
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.8"
+				},
+				"nei": {
+					"type": "gaaTil",
+					"steg": "3.7"
+				}
+			}
+		},
+		{
+			"stegnr": "3.7",
+			"spm": "Ligg mekanismen på starten av testsida?",
+			"ht": "<p>Gjer ein visuell inspeksjon og sjekk om mekanismen er plassert nær starten av testsida.</p>\n<p><strong>Merk:</strong> Du skal ikkje måtte scrolle vekk frå starten av sida for å nå mekanismen.</p>",
+			"type": "jaNei",
+			"kilde": [
+				"G186"
+			],
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.8"
+				},
+				"nei": {
+					"type": "gaaTil",
+					"steg": "3.10"
+				}
+			}
+		},
+		{
+			"stegnr": "3.8",
+			"spm": "Aktiver mekanismen, og sjekk om innhaldet blir sett på pause, stoppa eller skjult?",
+			"ht": "<p>Aktiver mekanismen og sjekk at innhaldet faktisk blir sett på pause, stoppa eller skjult.</p>",
+			"type": "jaNei",
+			"kilde": [
+				"G186"
+			],
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.9"
+				},
+				"nei": {
+					"type": "gaaTil",
+					"steg": "3.10"
+				}
+			}
+		},
+		{
+			"stegnr": "3.9",
+			"spm": "Startar innhaldet å bevege seg, blinka eller rulle, av seg sjølv igjen, før det har gått 1 minutt?",
+			"ht": "<p>Av praktiske grunnar, avgrensar vi testen til 1 minutt.</p>",
+			"type": "jaNei",
+			"kilde": [
+				"F16"
+			],
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.10"
+				},
+				"nei": {
+					"type": "avslutt",
+					"fasit": "Ja",
+					"utfall": "Det er mogleg å pause, stoppe eller skjule innhald som automatisk bevegar seg, blinkar eller rullar."
+				}
+			}
+		},
+		{
+			"stegnr": "3.10",
+			"spm": "Står det på testsida at det er mogleg å stoppa, pause eller skjula innhaldet som beveger seg, blinkar eller rullar, ved å bruke ein tastatursnarveg?",
+			"ht": "<p>Dokumentasjon av tastatursnarvegen skal anten ligge på starten av nettsida, eller i direkte tilknytning til det aktuelle innhaldet. Ein tastatursnarveg kan for eksempel vere:</p>\n<ul>\n<li>Esc-tasten</li>\n<li>Ein kombinasjon av Ctrl og ein annan tast </li>\n</ul>\n<p>Det viktige er at brukaren får informasjon om snarvegen og kva den gjer. </p>",
+			"type": "jaNei",
+			"kilde": [
+				"G4",
+				"G187"
+			],
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.11"
+				},
+				"nei": {
+					"type": "gaaTil",
+					"steg": "3.13"
+				}
+			}
+		},
+		{
+			"stegnr": "3.11",
+			"spm": "Aktiver tastatursnarvegen, og sjekk om innhaldet blir sett på pause, stoppa eller skjult.",
```

Comment:
Samsvar spørsmål og svaralternativ. For eksempel: "Vert innhaldet sett på pause, stoppa eller skjult når tastatursnarvegen aktiverast?"